cmd/dbfactory/cluster.go

```diff
@@ -65,6 +66,7 @@ var clusterProvisionCmd = &cobra.Command{
 		replicas, _ := command.Flags().GetString("replicas")
 		dbProxy, _ := command.Flags().GetBool("dbproxy")
 		creationSnapshotARN, _ := command.Flags().GetString("creation-snapshot-arn")
+		enableDevopsGuru, _ := command.Flags().GetBool("devopsguru")
```

Comment:
Let's do `devops-guru` instead of `devopsguru`

Reply:
+1 on this ;)